webpack.config.js

```diff
@@ -61,6 +62,14 @@ module.exports = {
 		new RtlCssPlugin( {
 			filename: '[name]-rtl.css',
 		} ),
+		new CopyPlugin( {
+			patterns: [
+				{
+					from: path.resolve( process.cwd(), 'assets', 'testimonials' ),
+					to: '[path]/images/[name][ext]',
+				},
+			],
+		} ),
```

Comment:
Do you mind explaining what this plugin is needed for? It seems to be related to the images but I am not positive why this is needed. I see that the `.png` assets that you are using are intended for block styles. For a point of reference for you, we have existing precedence of setting the images with the (core mechanism) block styles.

https://github.com/godaddy-wordpress/coblocks/blob/f190e7e7922a95e35e583a8e55ffd3ae1b42c272/src/blocks/opentable/styles/editor.scss#L9